ckcp/hack/util/reset_ckcp.sh

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Pipeline Service Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SCRIPT_DIR="$(
+  cd "$(dirname "$0")" >/dev/null || exit 1
+  pwd
+)"
+
+PLNSVC_DIR="$(dirname "$(dirname "$(dirname "$SCRIPT_DIR")")")"
```

Comment:
`PLNSVC_DIR` is usually called `PROJECT_DIR` in other scripts.
I would simplify to `"$(cd "$SCRIPT_DIR/../../.." >/dev/null || exit 1; pwd)"`

Reply:
@Roming22 a while back, one of the jobs was flaky because CI was unable to check ../.. structure. That's why I feel its better to use dirname.